packages/client/tests/functional/_utils/providers.ts

```diff
@@ -0,0 +1,9 @@
+export enum Providers {
```

Comment:
Do we test SQLSERVER and MARIADB?

Reply:
Not sure about mariadb, but we have a couple of sql-server-specific tests in old setup.